docs/monads.md

```diff
@@ -29,11 +29,11 @@ article.
 
 ## Intrinsic effects and capabilities
 
-We classify the effects into two categories:
+We classify effects into two categories:
 
 * _intrinsic effects_ provide operations enabled by the additional structure of
   the monad itself. For example, the intrinsic effect of `StateT` is adding
-  state to a computation, `ExceptT` adds alternative exit path, `ConduitM` adds
+  state to a computation, `ExceptT` adds the alternative exit path, `ConduitM` adds
```

Comment:
I believe the article should be indefinite here, since this is where this path is introduced for the first time.